dataflow/src/ops/distinct.rs

```diff
@@ -0,0 +1,252 @@
+use std::collections::HashMap;
+
+use prelude::*;
+
+/// This will get Distinct elements from each group
+/// TODO
+
```

Comment:
It' be great if you could expand this comment to explain what the operator does, how, and what it's limitations are :) In Rust, you also have to not leave a space between a documentation comment and the item it is documenting (i.e., the newline above has to go away).